azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/VirtualMachineInstanceView.java

```diff
@@ -2,335 +2,102 @@
-    private Integer platformUpdateDomain;
-
-    /**
-     * Specifies the fault domain of the virtual machine.
-     */
-    @JsonProperty(value = "platformFaultDomain")
-    private Integer platformFaultDomain;
-
-    /**
-     * The computer name assigned to the virtual machine.
-     */
-    @JsonProperty(value = "computerName")
-    private String computerName;
-
-    /**
-     * The Operating System running on the virtual machine.
-     */
-    @JsonProperty(value = "osName")
-    private String osName;
-
-    /**
-     * The version of Operating System running on the virtual machine.
-     */
-    @JsonProperty(value = "osVersion")
-    private String osVersion;
-
-    /**
-     * The Remote desktop certificate thumbprint.
-     */
-    @JsonProperty(value = "rdpThumbPrint")
-    private String rdpThumbPrint;
-
-    /**
-     * The VM Agent running on the virtual machine.
-     */
-    @JsonProperty(value = "vmAgent")
-    private VirtualMachineAgentInstanceView vmAgent;
-
-    /**
-     * The Maintenance Operation status on the virtual machine.
-     */
-    @JsonProperty(value = "maintenanceRedeployStatus")
-    private MaintenanceRedeployStatus maintenanceRedeployStatus;
-
-    /**
-     * The virtual machine disk information.
-     */
-    @JsonProperty(value = "disks")
-    private List<DiskInstanceView> disks;
-
-    /**
-     * The extensions information.
-     */
-    @JsonProperty(value = "extensions")
-    private List<VirtualMachineExtensionInstanceView> extensions;
-
-    /**
-     * Boot Diagnostics is a debugging feature which allows you to view Console
-     * Output and Screenshot to diagnose VM status. &lt;br&gt;&lt;br&gt; For
-     * Linux Virtual Machines, you can easily view the output of your console
-     * log. &lt;br&gt;&lt;br&gt; For both Windows and Linux virtual machines,
-     * Azure also enables you to see a screenshot of the VM from the
-     * hypervisor.
-     */
-    @JsonProperty(value = "bootDiagnostics")
-    private BootDiagnosticsInstanceView bootDiagnostics;
-
-    /**
-     * The resource status information.
-     */
-    @JsonProperty(value = "statuses")
-    private List<InstanceViewStatus> statuses;
-
+@Fluent
+public interface VirtualMachineInstanceView extends HasInner<VirtualMachineInstanceViewInner> {
     /**
      * Get specifies the update domain of the virtual machine.
      *
      * @return the platformUpdateDomain value
      */
-    public Integer platformUpdateDomain() {
-        return this.platformUpdateDomain;
-    }
-
-    /**
-     * Set specifies the update domain of the virtual machine.
-     *
-     * @param platformUpdateDomain the platformUpdateDomain value to set
-     * @return the VirtualMachineInstanceView object itself.
-     */
-    public VirtualMachineInstanceView withPlatformUpdateDomain(Integer platformUpdateDomain) {
-        this.platformUpdateDomain = platformUpdateDomain;
-        return this;
-    }
+    Integer platformUpdateDomain();
```

Comment:
Don't return boxed types for interfaces. Return a default integer if null.

Reply:
done